components/Scrollytelling/Scrollytelling.styles.ts

```diff
@@ -23,15 +23,14 @@ export type OverlayType = keyof typeof overlays;
 
 export const wrapper = 'relative';
 export const imageWrapper = 'sticky top-0 h-screen w-full z-0';
-export const image = 'hidden md:block absolute w-full h-full object-cover top-0 left-0 z-0';
-export const imageMobile = 'md:hidden absolute w-full h-full object-cover top-0 left-0 z-0';
+export const image = 'absolute w-full h-full object-cover top-0 left-0 z-0';
+// export const imageMobile = 'md:hidden absolute w-full h-full object-cover top-0 left-0 z-0';
```

Comment:
If the plan is to not use this then just remove the line of code. You can always come back and look at the git history to find it again.

Reply:
Oh this is my bad - I meant to remove this one